myna/core/workflow/config.py

```diff
@@ -136,6 +136,12 @@ def config(input_file, output_file=None, show_avail=False, overwrite=False):
                 step_obj.input_template = step_obj_prev.output_template
         step_obj.output_template = step.get(step_name).get("output_template")
 
+        # Get any CUI labels for the build
+        try:
+            settings["data"]["build"]["cui-markings"] = datatype.get_cui_info()
+        except NotImplementedError:
+            pass
```

Comment:
I figure this will be on a database-specific basis and that if it isn't implemented for a database than it is assumed to be unimportant.

It should still throw an exception if the function *is* implemented for the database class and something throws an exception other than a `NotImplementedError`